scripts/zones/Mhaura/IDs.lua

```diff
@@ -33,6 +33,7 @@ zones[xi.zone.MHAURA] =
         GOLDSMITHING_GUILD            = 7175, -- Everything you need for your goldsmithing needs!
         SMITHING_GUILD                = 7176, -- Welcome to the Blacksmiths' Guild salesroom!
         RAMUH_UNLOCKED                = 7389, -- You are now able to summon [Ifrit/Titan/Leviathan/Garuda/Shiva/Ramuh].
+        FYI_HERE_YOU_GO               = 7596, -- Here you go.  Sorry that I couldn't ship it to your Mog House
```

Comment:
That double-space looks suspicious, did you take this straight from client rips? (see my UpdateExtractor repo)

Reply:
I had no idea that existed -- but no, my comment was just typed in.  I assumed it was just for reference.  However: Your list is showing 7593 as the text that matches what I have for 7596 (which aligns to my capture).  I can update the comment text easy enough, but what should I do with numbers?

Reply:
My caps are from August, so they might be numerically a bit behind, as long as the content matches up with your recent caps that's fine

Reply:
Got it.  This should now be good to go!